collector/filesystem_common.go

```diff
@@ -56,7 +63,8 @@ func init() {
 // Filesystems stats.
 func NewFilesystemCollector() (Collector, error) {
 	subsystem := "filesystem"
-	pattern := regexp.MustCompile(*ignoredMountPoints)
+	mountPointPattern := regexp.MustCompile(*ignoredMountPoints)
+	FSTypesPattern := regexp.MustCompile(*ignoredFSTypes)
```

Comment:
Capitalized identifiers are exported.  Please make this `fsTypesPattern` or `filesystemTypesPattern`.